libs/common/package.json

```diff
@@ -29,7 +29,7 @@
     "unit-test-dev": "jest --verbose --config=unit.jest.config.js"
   },
   "dependencies": {
-    "mongodb": "^6.5.0",
+    "mongodb": "^4.17.0",
```

Comment:
Why has this been rolled back?

Reply:
Other services (except mock data loader & e2e-tests) use 4.17. This caused an issue that the `ObjectIds` are different - I only noticed when trying something like `expect(userInformation.id).toBe(expect.any(ObjectId))`